server/settings/settings.go

```diff
@@ -123,6 +123,7 @@ func (s *Server) Get(ctx context.Context, _ *settingspkg.SettingsQuery) (*settin
 		set.UiBannerURL = argoCDSettings.UiBannerURL
 		set.UiBannerPermanent = argoCDSettings.UiBannerPermanent
 		set.UiBannerPosition = argoCDSettings.UiBannerPosition
+		set.ServerCommitSummaryEnabled = argoCDSettings.ServerCommitSummaryEnabled
```

Comment:
Since we are in the context of a server settings here, no need to prepend "Server" to the name, but no big deal.